translations/binaries/slps_strings_food2.json

```
@@ -75,49 +75,49 @@
             "string": "まんぼうの串焼き[END]",
             "original_bytes_length": 17,
             "available_bytes_size": 23,
-            "translation": "망둥어 꼬치구이[END]"
+            "translation": "개복치 꼬치구이[END]"
         },
         {
             "offset": 1816304,
             "string": "魚鍋[END]",
             "original_bytes_length": 5,
             "available_bytes_size": 7,
-            "translation": "생선전골[END]"
+            "translation": "생선 전골[END]"
         },
         {
             "offset": 1816312,
             "string": "肉鍋[END]",
             "original_bytes_length": 5,
             "available_bytes_size": 7,
-            "translation": "육전골[END]"
+            "translation": "고기 전골[END]"
         },
         {
             "offset": 1816320,
             "string": "フルーツとうふ[END]",
             "original_bytes_length": 15,
             "available_bytes_size": 15,
-            "translation": "과일 두부[END]"
+            "translation": "프루츠 두부[END]"
         },
         {
             "offset": 1816336,
             "string": "フルーツチーズ[END]",
             "original_bytes_length": 15,
             "available_bytes_size": 15,
-            "translation": "과일 치즈[END]"
+            "translation": "프루츠 치즈[END]"
         },
         {
             "offset": 1816352,
             "string": "フルーツパフェ[END]",
             "original_bytes_length": 15,
             "available_bytes_size": 15,
-            "translation": "과일 파르페[END]"
+            "translation": "프루츠 파르페[END]"
```

```suggestion
            "translation": "과일 파르페[END]"
```

'Fruit'는 과일로 표기하는 게 자연스럽지 않을까요?

일본 여행 블로그/가이드 등에서 일본 상표나 메뉴 표기를 그대로 음차하느라 '후르츠 파르페'를 많이 쓰는 경항이 있지만, 국내에서 지칭할 때는 '과일 파르페'를 많이 쓰는 것 같습니다.